src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/InlineData.cs

```diff
@@ -8,60 +9,11 @@ namespace Xamarin.Android.Build.Tests
 {
 	static class InlineData
```

Comment:
Bit of a nit pick, but should this still be called InLineData? cos its not really inline its embedded ?